src/components/TransactionViewSidebarPanel/TransactionViewSidebarPanel.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './TransactionViewSidebarPanel.scss';
+import { Button } from 'carbon-components-react';
+
+type SidebarPanelProps = {
+    panelType: 'buttons' | 'filters' | 'log'
+};
+
+type SidebarPanelState = {
+    panelType: 'buttons' | 'filters' | 'log'
+};
+
+class TransactionViewSidebarPanel extends Component<SidebarPanelProps, SidebarPanelState> {
+
+    constructor(props: Readonly<SidebarPanelProps>) {
+        super(props);
+        this.state = {
+            panelType: this.props.panelType
+        }
+    }
+
+    public createTxn() {
+        console.log('create a new transaction');
+    }
+
+    public importTxn() {
+        console.log('import an existing transaction');
+    }
+
+    private getContents(): any {
+        let panelTSX: any;
+
+        if (this.state.panelType === 'buttons') {
+            panelTSX = (
+                <div className="panel-container" id="panel-container">
```

Comment:
Spotted it in a few places - I don't think you want an id & a class. I think you could probably remove these ids.